scripts/filter_language_issues.py

```
@@ -0,0 +1,181 @@
+        "typescript",
+        "python",
+        "dockerfile",
+        "kubernetes",
+        "docker",
+        "containerization",
+        "microservices",
+    }
+    return text.lower().strip() in technical_terms
+
+
+def should_ignore_rule(rule_id: str, message: str, text: str) -> bool:
```

Ruff (0.12.2)

Unused function argument: `message`

(ARG001)